webapp/src/components/sidebar_right/index.jsx

```diff
@@ -38,10 +38,10 @@ function mapPrsToDetails(prs, details) {
 function mapStateToProps(state) {
     return {
         username: state[`plugins-${pluginId}`].username,
-        reviews: mapPrsToDetails(state[`plugins-${pluginId}`].reviews, state[`plugins-${pluginId}`].reviewsDetails),
-        yourPrs: mapPrsToDetails(state[`plugins-${pluginId}`].yourPrs, state[`plugins-${pluginId}`].yourPrsDetails),
-        yourAssignments: state[`plugins-${pluginId}`].yourAssignments,
-        unreads: state[`plugins-${pluginId}`].unreads,
+        reviews: mapPrsToDetails(state[`plugins-${pluginId}`].sidebarContent.reviews, state[`plugins-${pluginId}`].reviewsDetails),
+        yourPrs: mapPrsToDetails(state[`plugins-${pluginId}`].sidebarContent.prs, state[`plugins-${pluginId}`].yourPrsDetails),
+        yourAssignments: state[`plugins-${pluginId}`].sidebarContent.assignments,
+        unreads: state[`plugins-${pluginId}`].sidebarContent.unreads,
```

Comment:
I'm actually thinking we should handle the "empty state" issue here, rather than the child component. Like this:

```suggestion
        reviews: mapPrsToDetails(state[`plugins-${pluginId}`].sidebarContent.reviews || emptyArray, state[`plugins-${pluginId}`].reviewsDetails),
        yourPrs: mapPrsToDetails(state[`plugins-${pluginId}`].sidebarContent.prs || emptyArray, state[`plugins-${pluginId}`].yourPrsDetails),
        yourAssignments: state[`plugins-${pluginId}`].sidebarContent.assignments || emptyArray,
        unreads: state[`plugins-${pluginId}`].sidebarContent.unreads || emptyArray,
```

Reply:
Also, `mapPrsToDetails` should be made into a proper selector with `reselect.createSelector` so we don't recompute the value every render